database/migration/migrateData.ts

```diff
@@ -892,10 +971,12 @@ const migrateData = async () => {
 // For running migrationScript.ts manually using `yarn migrate`
 if (isManualMigration) {
   console.log('Running migration script...')
-  migrateData().then(() => {
-    console.log('Done!\n')
-    process.exit(0)
-  })
+  migrateData()
+    .then(() => {
+      console.log('Done!\n')
+      process.exit(0)
+    })
+    .catch((e) => console.log('Error while migrating', e))
```

Comment:
I had a problem with migration, thought it was an error, but didn't realise i need to restart server before re-loading snapshot, while diagnosing adding this, can remove, but I think it doesn't hurt having it here ?

Reply:
It's designed to work two ways:
- you can manually run migration from the command line with `yarn migrate <version>`, (that's `isManualMigration`)
- it runs every time when the server starts (and checks for current version in DB)

I think you probably only needed to restart the server due to the `updateRowPolicies` error I mentioned above.